### PR TITLE
NFC - minor spelling tweaks under python directory

### PR DESCRIPTION
**Pull request by kiszk**
This PR addresses minor spelling tweaks under `tensorflow/python` directory.   
follow-on of #34958

### CODE REVIEW

**Comment by kiszk** [comment]
cc @mihaimaruseac

**Comment by kiszk** [comment]
I will try to create multiple PRs for `lite`, `core`, and others directories.

**Comment by jaingaurav** [comment]
@kiszk: I don't think that'll be necessary. We can review this as one bulk PR.

**Review by jaingaurav — DISMISSED** [review]

**Review by jaingaurav — DISMISSED** [review]

**Review by mdanatg — DISMISSED** [review]
Nice!

**Review by caisq — DISMISSED** [review]
LGTM for tensorflow/python/debug

**Review by mihaimaruseac — DISMISSED** [review]
Awesome. Thank you

**Review by alextp — DISMISSED** [review]

**Comment by gbaned** [comment]
@kiszk Can you please resolve conflicts? Thanks!

**Comment by kiszk** [comment]
Sure, I will address the conflict.

**Comment by kiszk** [comment]
@jaingaurav I am afraid that one big PR may increase the possibility of the conflict. What do you think?

**Comment by kiszk** [comment]
@gbaned I have just resolved conflict

**Comment by jaingaurav** [comment]
> @jaingaurav I am afraid that one future big PR (currently more than 400 files) may increase the possibility of the conflict. What do you think?

Whatever you think is best. Either way we can make sure we get all the necessary owner approvals.

**Review by mihaimaruseac — DISMISSED** [review]

**Comment by kiszk** [comment]
@jaingaurav I have just created three PRs (#35285, #35286, and #35287). They are all on my side now.

**Comment by enzopolo** [comment]
El 10 dic. 2019 07:05, "Kazuaki Ishizaki" <notifications@github.com>
escribió:
>
> This PR addresses minor spelling tweaks under tensorflow/python directory.
> follow-on of #34958
>
> ________________________________
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/tensorflow/tensorflow/pull/34985
>
> Commit Summary
> minor spelling tweaks
> File Changes
> M tensorflow/python/autograph/converters/conditional_expressions.py (2)
> M tensorflow/python/autograph/converters/logical_expressions.py (2)
> M tensorflow/python/autograph/converters/return_statements.py (2)
> M tensorflow/python/autograph/converters/return_statements_test.py (2)
> M tensorflow/python/autograph/core/converter.py (4)
> M tensorflow/python/autograph/core/function_wrappers.py (2)
> M tensorflow/python/autograph/core/function_wrappers_test.py (2)
> M tensorflow/python/autograph/core/naming.py (2)
> M tensorflow/python/autograph/impl/api.py (2)
> M tensorflow/python/autograph/impl/api_test.py (2)
> M tensorflow/python/autograph/impl/conversion.py (4)
> M tensorflow/python/autograph/operators/control_flow.py (6)
> M tensorflow/python/autograph/operators/py_builtins.py (6)
> M tensorflow/python/autograph/pyct/cfg.py (2)
> M tensorflow/python/autograph/pyct/common_transformers/anf_test.py (4)
> M tensorflow/python/autograph/pyct/origin_info.py (2)
> M tensorflow/python/autograph/pyct/parser.py (2)
> M tensorflow/python/autograph/pyct/static_analysis/activity.py (2)
> M
tensorflow/python/autograph/pyct/static_analysis/reaching_definitions.py (2)
> M tensorflow/python/autograph/pyct/transformer.py (2)
> M tensorflow/python/client/virtual_gpu_test.py (2)
> M tensorflow/python/compat/compat.py (2)
> M tensorflow/python/compat/compat_test.py (2)
> M tensorflow/python/compat/disable_v2_behavior_test.py (2)
> M tensorflow/python/compiler/tensorrt/test/biasadd_matmul_test.py (2)
> M tensorflow/python/compiler/tensorrt/test/dynamic_input_shapes_test.py
(2)
> M tensorflow/python/compiler/tensorrt/test/identity_output_test.py (2)
> M tensorflow/python/compiler/tensorrt/test/int32_test.py (2)
> M tensorflow/python/compiler/tensorrt/trt_convert.py (4)
> M tensorflow/python/compiler/tensorrt/trt_convert_test.py (2)
> M tensorflow/python/data/benchmarks/map_benchmark.py (2)
> M
tensorflow/python/data/experimental/kernel_tests/bucket_by_sequence_length_test.py
(2)
> M tensorflow/python/data/experimental/ops/readers.py (2)
> M tensorflow/python/data/kernel_tests/from_generator_test.py (2)
> M tensorflow/python/data/ops/dataset_ops.py (6)
> M tensorflow/python/data/util/sparse.py (2)
> M tensorflow/python/data/util/structure_test.py (2)
> M tensorflow/python/debug/cli/curses_ui.py (2)
> M tensorflow/python/debug/cli/debugger_cli_common.py (2)
> M tensorflow/python/debug/cli/debugger_cli_common_test.py (8)
> M tensorflow/python/debug/cli/evaluator.py (2)
> M tensorflow/python/debug/cli/profile_analyzer_cli_test.py (4)
> M tensorflow/python/debug/cli/tensor_format_test.py (2)
> M tensorflow/python/debug/lib/check_numerics_callback.py (2)
> M tensorflow/python/debug/lib/check_numerics_callback_test.py (6)
> M tensorflow/python/debug/lib/debug_data.py (2)
> M tensorflow/python/debug/lib/debug_gradients.py (6)
> M tensorflow/python/debug/lib/debug_graph_reconstruction_test.py (2)
> M tensorflow/python/debug/lib/debug_v2_ops_test.py (2)
> M tensorflow/python/debug/lib/dumping_callback.py (2)
> M tensorflow/python/debug/lib/dumping_callback_test.py (2)
> M tensorflow/python/debug/lib/dumping_callback_test_lib.py (4)
> M tensorflow/python/debug/lib/session_debug_grpc_test.py (4)
> M tensorflow/python/debug/lib/source_remote.py (2)
> M tensorflow/python/debug/lib/source_utils.py (2)
> M tensorflow/python/debug/lib/source_utils_test.py (2)
> M tensorflow/python/debug/wrappers/dumping_wrapper.py (2)
> M tensorflow/python/debug/wrappers/framework.py (4)
> M tensorflow/python/debug/wrappers/local_cli_wrapper.py (2)
> M tensorflow/python/debug/wrappers/local_cli_wrapper_test.py (2)
> M tensorflow/python/distribute/all_reduce.py (2)
> M tensorflow/python/distribute/cluster_resolver/cloud_tpu_client.py (2)
> M tensorflow/python/distribute/cluster_resolver/slurm_cluster_resolver.py
(4)
> M tensorflow/python/distribute/collective_all_reduce_strategy.py (4)
> M tensorflow/python/distribute/combinations.py (4)
> M tensorflow/python/distribute/cross_device_ops.py (2)
> M tensorflow/python/distribute/cross_device_utils.py (2)
> M tensorflow/python/distribute/device_util.py (2)
> M tensorflow/python/distribute/distribute_coordinator.py (8)
> M tensorflow/python/distribute/distribute_coordinator_test.py (4)
> M tensorflow/python/distribute/distribute_lib.py (2)
> M tensorflow/python/distribute/estimator_training.py (2)
> M tensorflow/python/distribute/input_lib.py (2)
> M tensorflow/python/distribute/mirrored_strategy.py (2)
> M tensorflow/python/distribute/mirrored_strategy_test.py (2)
> M tensorflow/python/distribute/model_collection/model_collection_base.py
(2)
> M tensorflow/python/distribute/model_collection/simple_models.py (2)
> M tensorflow/python/distribute/parameter_server_strategy.py (2)
> M tensorflow/python/distribute/parameter_server_strategy_test.py (4)
> M tensorflow/python/distribute/reduce_util.py (2)
> M tensorflow/python/distribute/saved_model_test_base.py (2)
> M tensorflow/python/distribute/values_test.py (2)
> M tensorflow/python/eager/benchmarks_test.py (2)
> M tensorflow/python/eager/def_function.py (6)
> M tensorflow/python/eager/forwardprop_test.py (2)
> M tensorflow/python/eager/forwardprop_util.py (2)
> M tensorflow/python/eager/function.py (16)
> M tensorflow/python/eager/function_test.py (2)
> M tensorflow/python/eager/pywrap_tensor.cc (8)
> M tensorflow/python/eager/pywrap_tfe.h (4)
> M tensorflow/python/eager/pywrap_tfe_src.cc (2)
> M tensorflow/python/eager/tape.py (2)
> M tensorflow/python/eager/wrap_function.py (6)
> M tensorflow/python/feature_column/feature_column_test.py (6)
> M tensorflow/python/feature_column/feature_column_v2.py (2)
> M tensorflow/python/feature_column/feature_column_v2_test.py (6)
> M tensorflow/python/feature_column/serialization.py (2)
> M tensorflow/python/framework/config_test.py (2)
> M tensorflow/python/framework/convert_to_constants.py (2)
> M tensorflow/python/framework/device_spec.py (4)
> M tensorflow/python/framework/func_graph.py (4)
> M tensorflow/python/framework/function.py (2)
> M tensorflow/python/framework/function_def_to_graph.py (2)
> M tensorflow/python/framework/graph_util_impl.py (2)
> M tensorflow/python/framework/meta_graph.py (2)
> M tensorflow/python/framework/op_callbacks.py (2)
> M tensorflow/python/framework/op_callbacks_test.py (8)
> M tensorflow/python/framework/op_def_library_test.py (2)
> M tensorflow/python/framework/ops.py (12)
> M tensorflow/python/framework/random_seed.py (2)
> M tensorflow/python/framework/smart_cond.py (2)
> M tensorflow/python/framework/subscribe.py (2)
> M tensorflow/python/framework/tensor_like.py (2)
> M tensorflow/python/framework/tensor_shape.py (2)
> M tensorflow/python/framework/tensor_shape_test.py (4)
> M tensorflow/python/framework/tensor_util.py (2)
> M tensorflow/python/framework/test_combinations.py (2)
> M tensorflow/python/framework/test_util.py (8)
> M tensorflow/python/framework/test_util_test.py (8)
> M tensorflow/python/framework/type_spec.py (4)
> M tensorflow/python/grappler/cost_analyzer.cc (4)
> M tensorflow/python/grappler/item_test.py (2)
> M tensorflow/python/keras/applications/applications_test.py (2)
> M tensorflow/python/keras/backend_config_test.py (2)
> M tensorflow/python/keras/distribute/distributed_training_utils.py (4)
> M tensorflow/python/keras/distribute/keras_correctness_test_base.py (6)
> M tensorflow/python/keras/distribute/multi_worker_callback_tf1_test.py (2)
> M tensorflow/python/keras/distribute/multi_worker_fault_tolerance_test.py
(8)
> M tensorflow/python/keras/engine/base_layer.py (2)
> M tensorflow/python/keras/engine/data_adapter.py (4)
> M tensorflow/python/keras/engine/network.py (2)
> M tensorflow/python/keras/engine/network_test.py (4)
> M tensorflow/python/keras/engine/training.py (4)
> M tensorflow/python/keras/engine/training_distributed.py (2)
> M tensorflow/python/keras/engine/training_utils.py (4)
> M tensorflow/python/keras/engine/training_v1.py (6)
> M tensorflow/python/keras/estimator/__init__.py (8)
> M tensorflow/python/keras/integration_test.py (2)
> M tensorflow/python/keras/layers/normalization.py (4)
> M tensorflow/python/keras/layers/preprocessing/image_preprocessing.py (2)
> M tensorflow/python/keras/layers/preprocessing/text_vectorization.py (8)
> M tensorflow/python/keras/layers/preprocessing/text_vectorization_test.py
(2)
> M tensorflow/python/keras/layers/recurrent.py (2)
> M tensorflow/python/keras/layers/recurrent_test.py (4)
> M tensorflow/python/keras/layers/recurrent_v2.py (2)
> M tensorflow/python/keras/layers/rnn_cell_wrapper_v2_test.py (2)
> M tensorflow/python/keras/layers/serialization_test.py (2)
> M tensorflow/python/keras/metrics.py (4)
> M tensorflow/python/keras/optimizer_v2/optimizer_v2.py (4)
> M tensorflow/python/keras/optimizer_v2/optimizer_v2_test.py (2)
> M tensorflow/python/keras/premade/wide_deep.py (2)
> M tensorflow/python/keras/saving/hdf5_format_test.py (2)
> M tensorflow/python/keras/saving/saved_model/save_impl.py (2)
> M tensorflow/python/keras/saving/saved_model/saved_model_test.py (2)
> M tensorflow/python/keras/saving/saved_model/serialized_attributes.py (4)
> M tensorflow/python/keras/utils/conv_utils.py (2)
> M tensorflow/python/keras/utils/io_utils_test.py (2)
> M tensorflow/python/keras/utils/metrics_utils.py (2)
> M tensorflow/python/kernel_tests/bias_op_deterministic_test.py (2)
> M tensorflow/python/kernel_tests/boosted_trees/stats_ops_test.py (2)
> M tensorflow/python/kernel_tests/clip_ops_test.py (4)
> M tensorflow/python/kernel_tests/critical_section_test.py (2)
> M tensorflow/python/kernel_tests/distributions/special_math_test.py (4)
> M tensorflow/python/kernel_tests/eig_op_test.py (2)
> M tensorflow/python/kernel_tests/einsum_op_test.py (2)
> M tensorflow/python/kernel_tests/fractional_max_pool_op_test.py (4)
> M tensorflow/python/kernel_tests/linalg/linear_operator_diag_test.py (2)
> M
tensorflow/python/kernel_tests/linalg/linear_operator_full_matrix_test.py
(2)
> M
tensorflow/python/kernel_tests/linalg/linear_operator_low_rank_update_test.py
(2)
> M
tensorflow/python/kernel_tests/linalg/linear_operator_lower_triangular_test.py
(2)

**Comment by gbaned** [comment]
@kiszk Can you please resolve conflicts? Thanks!

**Comment by kiszk** [comment]
@gbaned Thank you for pinging me. I have just resolved the conflicts.

**Review by mihaimaruseac — DISMISSED** [review]

**Comment by gbaned** [comment]
@kiszk Can you please resolve conflicts? Thanks!

**Comment by kiszk** [comment]
@gbaned Sure, done again.

**Review by mihaimaruseac — DISMISSED** [review]

**Review by mihaimaruseac — DISMISSED** [review]

**Comment by gbaned** [comment]
@kiszk Can you please resolve conflicts? Thanks!

**Comment by kiszk** [comment]
@gbaned Thank you for pinging me, done.

**Review by mihaimaruseac — DISMISSED** [review]

**Comment by kiszk** [comment]
@gbaned Just resolved the conflict, could you handle this?

**Comment by ravikyram** [comment]
@kiszk 

I see still some conflicts to be resolved.Can you please resolve conflicts? Thanks!

**Comment by kiszk** [comment]
@ravikyram Thank you. Just resolved the conflict, could you review this?

**Review by jaingaurav — DISMISSED** [review]

**Comment by kiszk** [comment]
Resolved conflict again.

**Review by jaingaurav — DISMISSED** [review]

**Comment by jaingaurav** [comment]
@kiszk: Seems like there are still some conflicts. Are you seeing any on your end?

**Comment by kiszk** [comment]
@jaingaurav. There is no conflict on github site. Also no conflict on my end.

**Comment by mihaimaruseac** [comment]
Manually imported it

**Comment by mihaimaruseac** [comment]
There are some conflicts to solve, please.

**Comment by kiszk** [comment]
@mihaimaruseac I see. Now, I saw a new conflict. Resolved, thank you for pinging me.

**Review by mihaimaruseac — DISMISSED** [review]

**Review by mihaimaruseac — DISMISSED** [review]

**Comment by mihaimaruseac** [comment]
There are new conflicts, can you please rebase on master again?

**Comment by kiszk** [comment]
@mihaimaruseac Just resolved conflicts, again

**Review by jaingaurav — APPROVED** [review]
